Testsuite/test/post_process_ctest_results.py

```diff
@@ -1,75 +1,126 @@
 import sys
-import io
 import re
 import os
 
-report_file=sys.argv[1]
-report_name=sys.argv[2]
-global_report_name=sys.argv[3]
-rx=re.compile('(.*Configuring (examples|demo|test)*( in )*(test\/|examples\/|demo\/)*)((?!done)\w+)')
-rx_demo=re.compile('.*in demo\/')
-rx_examples=re.compile('.*in examples\/')
+input_report_file_name=sys.argv[1]
+report_file_name=sys.argv[2]
+global_report_file_name=sys.argv[3]
+config_regex=re.compile('(.*Configuring (examples|demo|test)*( in )*(test\/|examples\/|demo\/)*)((?!done)\w+)')
+demo_regex=re.compile('.*in demo\/')
+examples_regex=re.compile('.*in examples\/')
```

Comment:
Please try with raw strings:
```suggestion
config_regex=re.compile(r'(.*Configuring (examples|demo|test)*( in )*(test/|examples/|demo/)*)((?!done)\w+)')
demo_regex=re.compile(r'.*in demo/')
examples_regex=re.compile(r'.*in examples/')

```